appeals/web/src/server/lib/mappers/appellantCase.mapper.js

```diff
@@ -113,6 +114,49 @@ export function initialiseAndMapData(appellantCaseData, currentRoute) {
 		inputItemApi: '#'
 	};
 
+	mappedData.agentName = {
+		id: 'agent-name',
+		display: {
+			summaryListItem: {
+				key: {
+					text: 'Agent name'
+				},
+				value: {
+					text: nameToString({
+						firstName: appealDetails.agent?.firstName || '',
+						lastName: appealDetails.agent?.firstName || ''
```

Comment:
I think this should be `lastName`, this must be what's causing the snapshot issue